app/views/content_items/get_involved.html.erb

```diff
@@ -36,17 +36,13 @@
       </div>
       <div class="consultation-lists govuk-grid-column-two-thirds-from-desktop">
         <div class="consultation-closing-soon">
-          <% @next_closing_consultations.each do |consultation| %>
-            <h3><%= time_until_closure(consultation) %></h3>
-          <% end %>
+          <h3><%= time_until_closure(@next_closing_consultation) %></h3>
           <ol class="document-list">
-            <% @next_closing_consultations.each do |consultation| %>
-              <%= content_tag(:li, "consultation", class: 'document-row') do %>
-                <h3><%= consultation['title'] %></h3>
-                <ul class="attributes">
-                  <li><%= link_to "Read and respond", consultation['base_path'], class: "govuk-link" %></li>
-                </ul>
-              <% end %>
+            <%= content_tag(:li, "consultation", class: 'document-row') do %>
```

Comment:
Maybe I've missed something, but it looks like this code previously showed multiple consultations, but now only shows one?  Is that intentional?

Reply:
In this case yes, this particular entry is for the consultation that explicitly shows on the top of the Get Involved page, in the blue box (which as of writing is an entry that closes tomorrow: https://www.gov.uk/government/get-involved )

Below on lines 54 and 72 are the entries for multiple consultations in their respective areas.

Reply:
Thanks for explaining!  I missed the `[0]` from `open_consults.sort_by! { |k| k['details']['closing_date'] }[0]` in the `retrieve_next_closing` method.  Seems strange that the code in Whitehall returned a single item from the method but then iterated over an array that always contained one item in the views 🤷‍♂️.  Good that this strange logic has been refactored and simplified here 👍.